org.uqbar.project.wollok.game.ui/src/org/uqbar/project/wollok/game/ui/exportWizards/ExportGamePage.java

```diff
@@ -0,0 +1,100 @@
+package org.uqbar.project.wollok.game.ui.exportWizards;
+
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.KeyEvent;
+import org.eclipse.swt.events.KeyListener;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.FileDialog;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Shell;
+import org.eclipse.swt.widgets.Text;
+
+public class ExportGamePage extends WizardPage{
+	 private Text text1;
+	  private Composite container;
+
+	  public ExportGamePage() {
+	    super("Export Wollok Game");
```

Comment:
i18n

If it is java we can automatically generate the properties file and the java class with the constants.
There's a refactor called "externalize strings" or something like that.
Check out our Messages class